service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -477,8 +503,9 @@ protected final ConnectionLimitManager getConnectionLimitManager() {
 
             log.info("using ResourceLimitChecks [{}]", resourceLimitChecks.getClass().getName());
 
+            startServiceClient(telemetrySender, "AMQP Messaging Network (Telemetry)");
+            startServiceClient(eventSender, "AMQP Messaging Network (Event)");
```

Comment:
I had already been wondering whether it would be useful to have completely separate implementations of TelemetrySender and EventSender. However, it would then be harder to share a single AMQP connection to the Messaging Network. WDYT?

Reply:
Ok, I see, that's a reason not to separate it then.